src/core/dev_api/openvino/core/tensor_util.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/constant.hpp"
+#include "ov_optional.hpp"
+
+namespace ov {
+namespace util {
+
+OPENVINO_API Tensor greater_equal(const ov::Tensor& lhs, const ov::Tensor& rhs);
+template <typename T>
+OPENVINO_API Tensor greater_equal(const ov::Tensor& lhs, const T& element);
+OPENVINO_API bool reduce_and(const ov::Tensor& t);
+template <typename T>
+OPENVINO_API ov::optional<std::vector<T>> to_vector(const ov::Tensor& t);
```

Comment:
Remove as it it not used anywhere?

Reply:
seems like it is useful for debugging and in transformations, so I'll keep it for now